### PR TITLE
IVR Fixes

### PR DESCRIPTION
**Pull request by ericnewcomer**
* Don't handle text messages in IVR flows (active flow case)
* Show IVR icons on contact page, uniform listen / stop buttons
* Fix messaging for missed calls / ivr messages on contact page

### CODE REVIEW

File: temba/flows/models.py [hunk 1]
```diff
@@ -593,6 +593,11 @@ def find_and_handle(cls, msg):
 
         for step in steps:
             flow = step.run.flow
+
+            # voice flows don't deal with text messages
+            if flow.flow_type == Flow.VOICE:
+                return False
```
Comment (nicpottier) [inline temba/flows/models.py]: Is this really what we want? Shouldn't msg flows still have the opportunity to deal with the message?

Seems like adding ```run__flow__flow_type=Flow.MESSAGE``` in the flow step filter above would be the more natural way to deal with this.

Comment (ericnewcomer) [inline temba/flows/models.py]: Hmmm.. ya, I guess that makes sense. It's all a bit weird in any case.

Comment (nicpottier) [inline temba/flows/models.py]: Your face is a bit weird.

**Comment by ericnewcomer** [comment]
ok, I think this is ready, want to merge for push tonight.

**Pull request merged**